lesson03.find-a-pair/index.html

```diff
@@ -1,22 +1,188 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Find A Pair</title>
-    <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
-
-        }
-    </style>
+    <title>Find a Pair</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="shelly Eldman">
+    <meta name="keywords" content="HTML,CSS,JavaScript">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
 </head>
+
 <body>
-    <header>
-        <h1>>Find A Pair</h1>
-    </header>
-    <main>
-        Content Here
-    </main>
+    <h1>Cards Memory Game</h1>
+
+    <div class="container">
+
+        <div class="card-wrap">
```

Comment:
That's a lot of HTML. how can you make it more efficient?
Think about how you would do it for 100 cards